### PR TITLE
run detail order/sort by finished column

### LINKED ISSUE #212: UI - Run detail - incorrect order of jobs
In RunDetail view, the order of jobs is not correct. ( should be order by order ) 
image

### PR DESCRIPTION
**Pull request by senelesithole**
fix #212

### CODE REVIEW

**Comment by kevinwallimann** [comment]
According to the issue, the jobs should be ordered by job order. If you sort it by started or finished time, the sorting will be undefined /random when the jobs haven't started yet.

**Review by jozefbakus — CHANGES_REQUESTED** [review]
I would solve this on effect side.
Response for job instances contains order field, but it is not defined in js class. 
So you have to add order: number into JobInstanceModel in jobInstance.model.ts.
And update runDetailGet in runs.effects.ts (sort instances that go into store somewhere here: 
`return [
            {
              type: RunActions.GET_DAG_RUN_DETAIL_SUCCESS,
              payload: jobInstance,
            },
          ];`)

File: ui/src/app/stores/runs/runs.effects.ts [hunk 1]
```diff
@@ -59,7 +59,7 @@ export class RunsEffects {
           return [
             {
               type: RunActions.GET_DAG_RUN_DETAIL_SUCCESS,
-              payload: jobInstance,
+              payload: jobInstance.sort((jobInstanceA, jobInstanceB) => jobInstanceA.order - jobInstanceB.order),
```
Comment (jozefbakus) [inline ui/src/app/stores/runs/runs.effects.ts:62]: Could you please write test for this piece.

File: ui/src/app/stores/runs/runs.effects.spec.ts [hunk 2]
```diff
@@ -90,24 +90,48 @@ describe('RunsEffects', () => {
...
+          new Date(Date.now()),
+          JobStatusFactory.create('Status'),
+          3,
         ),
       ];
 
       const action = new GetDagRunDetail(id);
       mockActions = cold('-a', { a: action });
-      const dagRunDetailResponse = cold('-a|', { a: jobInstances });
+      const dagRunDetailResponse = cold('-a|', { a: [jobInstancesA, jobInstancesB, jobInstancesC] });
```
Comment (jozefbakus) [inline ui/src/app/stores/runs/runs.effects.spec.ts]: Could you please change this to CBA ?

**Review by jozefbakus — APPROVED** [review]

**Pull request merged**